docs/generated/devkit/nrwl_devkit.md

```diff
@@ -150,6 +150,8 @@ It only uses language primitives and immutable objects
 - [detectPackageManager](../../devkit/documents/nrwl_devkit#detectpackagemanager)
 - [detectWorkspaceScope](../../devkit/documents/nrwl_devkit#detectworkspacescope)
 - [extractLayoutDirectory](../../devkit/documents/nrwl_devkit#extractlayoutdirectory)
+- [forEachExecutorOptions](../../devkit/documents/nrwl_devkit#foreachexecutoroptions)
```

Comment:
I agree it should be public api... It's super useful.

But this name is ugly (my fault) and I also don't know if I like the fact that theres 2 versions...

Can we hold off on making it public api for now?